sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -542,6 +542,12 @@ pi_result _pi_context::finalize() {
   return PI_SUCCESS;
 }
 
+bool _pi_queue::isInOrderQueue() const {
```

Comment:
I think this can be a static function with a pi_queue being passed in?

Reply:
sure, that can be done. could you please share why you would prefer static function here?

Reply:
It will avoid one level of indirection. But it's fine without that.